src/canvas-effects/canvas-effect.js

```diff
@@ -435,159 +437,61 @@ export default class CanvasEffect extends PIXI.Container {
 		);
 	}
 
-	/**
-	 * Getter for the current playing video of the effect
-	 *
-	 * @returns {null|*}
-	 */
-	get video() {
-		return this._video;
-	}
-
-	/**
-	 * Setter for the current playing video of the effect
-	 */
-	set video(inVideo) {
-		if (!inVideo) return;
-
-		inVideo.playbackRate = this.data.playbackRate
-			? this.data.playbackRate
-			: 1.0;
-		inVideo.muted = !this.data.volume;
-		inVideo.volume =
-			(this.data.volume ?? 0) *
-			game.settings.get("core", "globalInterfaceVolume");
-
-		if (!this._video) {
-			this._video = inVideo;
-			return;
-		}
-
-		const isLooping = this._video?.loop;
-		const currentTime = this._video.currentTime;
-
-		this._video = inVideo;
-
-		this._video.currentTime = Math.min(currentTime, this._video.duration);
-		this._video.loop = isLooping;
-
-		this.updateTexture();
-	}
-
 	async playMedia() {
-		if (this.animatedSprite) {
-			await this.sprite.play();
-		} else if (this.video) {
-			try {
-				await this.video.play().then(() => {
-					this.updateTexture();
-				});
-			} catch (err) {
-			}
+		if (this.destroyed || this._ended || this.isEnding) {
```

Comment:
`this.isEnding` getter has been removed, is this a property of `this.sprite` now?

Reply:
I probably just missed it, have to check again

Reply:
Hmm, `this.isEnding` is still there and has the same function as before. Am I missing something obvious here..?

Reply:
I think I just need to purge these references then :D